app/src/test/resources/application-test.yml

```diff
@@ -37,20 +37,4 @@ lhAPIProvider:
   vroAudURL: "${VRO_AUD_URL:placeholderUrl}"
   apiKey: "${LH_VRO_API_KEY:placeholderKey}"
   validateToken: "NO"
-bip:
-  claimClientId: "${BIP_CLAIM_USERID:claimuser}"
```

Comment:
Are these still required, aren't they?

Reply:
they they are moved to application-local. This ones inherits that one.

Reply:
Ok. I see.